rspace/src/main/scala/coop/rchain/rspace/ScodecModels.scala

```diff
@@ -0,0 +1,43 @@
+package coop.rchain.rspace.scodecmodels
```

Comment:
I'd like to put the contents of this file in the [`internal`](https://github.com/rchain/rchain/blob/5d47fb308a28b142741260a70e03fa5dc5644e98/rspace/src/main/scala/coop/rchain/rspace/internal.scala) object